feat(crm): Add some treatment for unidentified group actors

danielbachhuber opened the pull request:
See https://posthoghelp.zendesk.com/agent/tickets/28587

## Changes

Adds some treatment for unidentified group actors:

| Before | After |
|--------|--------|
| ![CleanShot 2025-04-07 at 16 42 15@2x](https://github.com/user-attachments/assets/5c965351-fbb8-483b-b15e-642dbcef5158) | ![CleanShot 2025-04-09 at 04 31 23@2x](https://github.com/user-attachments/assets/7b964c1a-bd09-4fdf-a3df-17bf8e6a481b) | 

## How did you test this code?

Visual review + tests for the backend bits.

danielbachhuber committed:
Add some treatment for unidentified group actors

danielbachhuber committed:
Show a more helpful message when properties aren't defined

danielbachhuber committed:
Prevent JavaScript error when `name` isn't present

danielbachhuber committed:
Add a test for bfb8615df66a68ef859d9f61436b452c5d0c2a70

github-actions[bot] committed:
Update UI snapshots for `chromium` (7)

github-actions[bot] committed:
Update UI snapshots for `chromium` (5)

github-actions[bot] committed:
Update UI snapshots for `chromium` (7)

github-actions[bot] committed:
Update UI snapshots for `chromium` (5)

github-actions[bot] committed:
Update UI snapshots for `chromium` (5)

github-actions[bot] committed:
Update UI snapshots for `chromium` (5)

thmsobrmlr reviewed (approved):
This is great!

Am I right assuming that this will also show merged and deleted groups in the same way? Might be nice to use a tooltip with that info and only there link to the page for creating groups if this is the case.

danielbachhuber committed:
Merge branch 'master' into crm/unidentified-groups

danielbachhuber committed:
Use tooltip instead of inline link

danielbachhuber committed:
Hide missing persons message for groups

danielbachhuber committed:
Tweak search placeholder for groups

danielbachhuber commented:
> Am I right assuming that this will also show merged and deleted groups in the same way?

Merged and deleted groups don't exist yet...

The scenario where you'd see this is when you've sent events with `$groups` but you haven't `$groupidentify` the group yet.

Good call out, though. I removed the notice with 7e15bd199e (seems confusing when the groups actually appear in the list)

> Might be nice to use a tooltip with that info and only there link to the page for creating groups if this is the case.

Ah, good suggestion, thank you. Updated with f093773525
